packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -1208,6 +1212,43 @@ const AccountSwitcher = props => {
     const real_account = props.is_pre_appstore ? traders_hub_real_accounts : default_real_accounts;
 
     const demo_accounts = props.is_pre_appstore ? traders_hub_demo_account : default_demo_accounts;
+    const first_real_login_id = props.account_list?.find(account => account.loginid?.[0]).loginid;
+
+    const TradersHubRedirect = () => {
+        const TradersHubLink = () => {
+            const handleRedirect = () => {
+                if (!props.is_virtual && isDemoAccountTab) {
```

Comment:
what is the expected beheviour for the other conditions rather than these ?

Reply:
It will still redirect them without switching.